deployment/common/proposalutils/mcms_helpers.go

```diff
@@ -165,15 +165,17 @@ func RunTimelockExecutor(env deployment.Environment, cfg RunTimelockExecutorConf
 				if err != nil {
 					return fmt.Errorf("error creating timelock executor proxy: %w", err)
 				}
+
+				ch := env.Chains[cfg.ChainSelector].DeployerKey
```

Comment:
@cedric-cordenier not sure if this was meant to be included

Reply:
Nope, definitely not meant to be included

Reply:
Removed!